tests/integration/test_charm.py

```diff
@@ -395,21 +393,57 @@ async def test_promote_user(
     act: when promoting a user
     assert: the user should be promoted successfully.
     """
+    with requests.session() as sess:
+        res = sess.get(f"{discourse_address}/session/csrf", headers={"Accept": "application/json"})
+        # pylint doesn't see the "ok" member
+        assert res.status_code == requests.codes.ok, res.text  # pylint: disable=no-member
```

Comment:
Why not res.ok?

Reply:
similar code can be found in conftest, i decided to keep it consistent with what already exists in the repo

Reply:
Makes sense, but I'd rather have it all changed incrementally

Reply:
i have changed everywhere to assert response.ok instead